pkg/events/events.go

```diff
@@ -3,7 +3,7 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // --------------------------------------------------------------------------------------------
 
-package k8scontext
+package events
 
 // EventType represents the type of update for k8s resources
```

Comment:
Need update to `EventType represents the type of update` ?

Reply:
Yes - let's clarify these comments!